Sources/Sentry/SentryANRTrackingIntegration.m

```diff
@@ -195,15 +195,85 @@ - (void)anrStoppedWithResult:(SentryANRStoppedResult *_Nullable)result
     [self.fileManager deleteAppHangEvent];
 
     // We round to 0.1 seconds accuracy because we can't precicely measure the app hand duration.
-    NSString *errorMessage =
-        [NSString stringWithFormat:@"App hanging between %.1f and %.1f seconds.",
-            result.minDuration, result.maxDuration];
-
+    NSString *appHangDurationInfo = [NSString
+        stringWithFormat:@"between %.1f and %.1f seconds", result.minDuration, result.maxDuration];
+    NSString *errorMessage = [NSString stringWithFormat:@"App hanging %@.", appHangDurationInfo];
     event.exceptions.firstObject.value = errorMessage;
-    [SentrySDK captureEvent:event];
+
+    NSMutableDictionary *mechanismData = [event.exceptions.firstObject.mechanism.data mutableCopy];
```

Comment:
`m`: `exceptions`, `firstObject`, `mechanism` and `data` are all nullable but used in chain here. Are we certain this will not cause any issues down the road without any null checks?

Reply:
Good catch, thanks. I added an extra check.